lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -298,6 +301,16 @@ def interface_network(libvirt_client, options)
             end
           end
 
+          if options[:ipv6_address]
+            address = options[:ipv6_address]
+            available_networks.each do |network|
+              if options[:ipv6_address] == network[:ip_address]
```

Comment:
This looks odd. It means the ipv6_address as specified should be the one of the host where the IPv4 address should be the one of the VM. I think it would be better to look if the `ipv6_address` is within the IPv6 network as is done in IPv4.